l10n_mx_cfdi/__manifest__.py

```diff
@@ -7,7 +7,7 @@
     "license": "LGPL-3",
     "category": "Accounting",
     "version": "17.0.1.0.0",
-    "depends": ["account", "l10n_mx_catalogs"],
+    "depends": ["l10n_mx_catalogs", "product"],
```

Comment:
Are we sure we don't have/want to depend on l10n_mx ?

Reply:
@azubieta `l10n_mx_catalogs` already depends on `l10n_mx`.